### DISCUSSION TITLE
Neutone on Mac OS 10.14

### DISCUSSION THREAD

**Discussion by Dremor412** [QosmoInc/neutone_sdk#62] [Q&A]
Hello ! Is there any chance Neutone plugin can be compatible with Mac OS 10.14 ? or anyway to turn Neutone model into .ts rave model ? 
Thanks for your fantastic work !

**Comment by Farshidbagheri** [comment]
Based on the current Neutone FX requirements, macOS 10.14 Mojave does not appear to be officially supported. The listed requirement is macOS 10.15+ for Intel Macs, and for newer releases Intel Mac support has also been discontinued from v1.6.1 onward, so Intel users may need v1.6.0 or earlier.

For the RAVE question, I do not think there is a direct or officially supported “Neutone model → RAVE .ts” conversion path. The usual workflow seems to be the opposite direction: train/export the RAVE model to TorchScript, then wrap/export it for use in Neutone through the Neutone SDK.

So practically:

1. If you only have a packaged Neutone model, converting it back into a RAVE `.ts` model is probably not supported.
2. If you have the original RAVE checkpoint/training folder, export it again using the RAVE export workflow.
3. If you specifically need RAVE format, the safest path is to train/export the model directly from RAVE rather than trying to reverse-convert a Neutone package.

I hope this helps clarify the compatibility and conversion direction.
